### PR TITLE
minor doc updates trying to run through README

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees  the minor doc updates we've discussed

### CODE REVIEW

File: docs/debugging-openshift.md [hunk 1]
```diff
@@ -113,6 +113,16 @@ If you simply want to take a container that OpenShift has created but debug it o
 
 Obviously this won't work if you don't have bash installed but you could always package it in for debugging purposes.
 
+Name Resolution Within Containers
+-------------------
+
+DNS related services like `dnsmasq` can interfere with naming resolution in the Docker containers launched by OpenShift, including binding on the same port (53) that OpenShift will attempt to use.  To circumvent this conflict, disable
+these services.  Using the `dnsmasq` example on Fedora, run all three of the following to ensure `dnsmasq` does not launch:
```
Comment (bparees) [inline docs/debugging-openshift.md]: should note that it needs to be done before starting openshift.

Comment (gabemontero) [inline docs/debugging-openshift.md]: just pushed update

**Comment by bparees** [comment]
lgtm 
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**